addons/mail_bot/static/tests/systray_messaging_menu_tests.js

```diff
@@ -169,6 +170,26 @@ QUnit.test('messaging menu widget: respond to notification prompt', function (as
     messagingMenu.destroy();
 });
 
+QUnit.test('notification_alert widget: display blocked notification alert', function (assert) {
+    assert.expect(2);
+
+    window.Notification.permission = 'denied';
+
+    var form = testUtils.createView({
+            View: FormView,
+            model: 'mail.message',
+            data: this.data,
+            arch: '<form>' +
+                '<widget name="notification_alert"/>' +
+            '</form>',
+    });
+    assert.containsOnce(form, '.o_widget .notification_alert', "Blocked notification alert should be displayed");
+    window.Notification.permission = 'granted';
+    form.reload();
+    assert.containsNone(form, '.o_widget .notification_alert', "Blocked notification alert should not be displayed");
+    form.destroy();
+});
+
```

Comment:
Remove this empty line (but keep a single empty line at the end of this test and the qunit module closing brackets).